### ISSUE TITLE
Logging output incorrect? 

### ISSUE THREAD

**Issue by gcmurphy** [securego/gosec#305]
### Summary 

Logging output is incorrect for package name: 

[gosec] 2019/04/27 08:10:10 Checking package: **command-line-arguments**

LOC: https://github.com/securego/gosec/blob/master/analyzer.go#L169

Potentially introduced by changes in 85d180848dbb8f86cba5d71bbbe06e1b40ce2705 

### Steps to reproduce the behavior

Run latest gosec build against any directory: 

```
./gosec .        
[gosec] 2019/04/27 08:10:09 including rules: default
[gosec] 2019/04/27 08:10:09 excluding rules: default
[gosec] 2019/04/27 08:10:09 Searching directory: /Users/gm/go/src/github.com/securego/gosec
[gosec] 2019/04/27 08:10:10 Checking package: command-line-arguments
[gosec] 2019/04/27 08:10:10 Checking file: /Users/gm/go/src/github.com/securego/gosec/analyzer.go
[gosec] 2019/04/27 08:10:10 Checking file: /Users/gm/go/src/github.com/securego/gosec/call_list.go
[gosec] 2019/04/27 08:10:10 Checking file: /Users/gm/go/src/github.com/securego/gosec/config.go
[gosec] 2019/04/27 08:10:10 Checking file: /Users/gm/go/src/github.com/securego/gosec/errors.go
[gosec] 2019/04/27 08:10:10 Checking file: /Users/gm/go/src/github.com/securego/gosec/helpers.go
[gosec] 2019/04/27 08:10:10 Checking file: /Users/gm/go/src/github.com/securego/gosec/import_tracker.go
[gosec] 2019/04/27 08:10:10 Checking file: /Users/gm/go/src/github.com/securego/gosec/issue.go
[gosec] 2019/04/27 08:10:10 Checking file: /Users/gm/go/src/github.com/securego/gosec/resolve.go
[gosec] 2019/04/27 08:10:10 Checking file: /Users/gm/go/src/github.com/securego/gosec/rule.go
Results:


Summary:
   Files: 9
   Lines: 1201
   Nosec: 4
  Issues: 0
```

### gosec version

```
▶ git describe --always --tags
1.3.0-12-g85d1808

▶ git rev-parse HEAD
85d180848dbb8f86cba5d71bbbe06e1b40ce2705
```

### Go version (output of 'go version')

```
▶ go version
go version go1.12.4 darwin/amd64
```

### Operating system / Environment

```
▶ uname -r
18.5.0
```
### Expected behavior

Show correct package name or don't log that information...

### Actual behavior

**command-line-argument**

**Comment by ccojocar** [comment]
There is a fix for this in this PR https://github.com/securego/gosec/pull/301/commits/5fd30dc2dbc06d8b8d887de9d20707450e1a1a9b

**Comment by ccojocar** [comment]
Fixed by #307
